compiler/parser/parser.peg

```diff
@@ -1190,9 +1190,12 @@ FieldName
 
 AndToken = ("and" / "AND") !IdentifierRest { RETURN("and") }
 OrToken = ("or" / "OR") !IdentifierRest { RETURN("or") }
-InToken = "in" !IdentifierRest { RETURN("in") }
 NotToken = ("not" / "NOT") !IdentifierRest { RETURN("not") }
-ByToken = "by" !IdentifierRest { RETURN("by") }
+ByToken = "by" !IdentifierRest
+InToken = "in" !IdentifierRest
+NullToken = "null" !IdentifierRest
+TrueToken = "true" !IdentifierRest
+FalseToken = "false" !IdentifierRest
```

Comment:
Maybe alphabetize these?

Reply:
Sure but I think it `true` and `false` should be grouped together.